### ISSUE TITLE
Regions outside US (latency)

### ISSUE THREAD

**Issue by 0x53A** [pulumi/console-requests#15]
Issues like https://github.com/pulumi/pulumi/issues/6445 and https://github.com/pulumi/pulumi/issues/5453 get massively exacerbated if you are outside the mainland US.

For example, I'm in Frankfurt and while the homepage (https://www.pulumi.com/) seems to be behind a worldwide CDN and requests take 15 to 55ms, the web app (https://app.pulumi.com/) takes >= 150ms per request.

The same is observable with API calls, like here: https://github.com/pulumi/pulumi-azure-native/issues/1030.

----------

Are there any plans to add more regions to your backend?

**Comment by chrsmith** [comment]
> Are there any plans to add more regions to your backend?

Yes! This is definitely something we've known about for a while and want to address. Though I cannot give you a firm timeline for when the Pulumi Service will be distributed across the world. Though I'd expect us to deliver some meaningful improvements in this area over the next couple of quarters.

As you noticed, www.pulumi.com is hosted from a CDN. But unfortunately our API service (api.pulumi.com) is a bit more tricky than just standing up new instances closer to users. For example, we'd want to upload your cloud resource data to location that's closer to you as well... but that opens up issues of configuration, policy, etc.

Anyways, just wanted to acknowledge the issue, apologize, and say we will solve this eventually.

**Comment by 0x53A** [comment]
question, if I were to create a VM in the US to be nearer to the pulumi servers, which Azure Region would be best?

I assume you are in SV / California, so ``West US``?

Thanks!
